hedera-node/hedera-mono-service/build.gradle.kts

```diff
@@ -82,9 +82,7 @@ java.sourceSets["main"].java.srcDir(generatedSources)
 tasks.processResources {
   filesMatching("semantic-version.properties") {
     filter { line: String ->
-      if (line.contains("hapi-proto.version")) {
-        "hapi.proto.version=" + libs.versions.com.hedera.hashgraph.protobuf.java.api.get()
-      } else if (line.contains("project.version")) {
```

Comment:
Not sure: Can this just be removed?

Reply:
No think it needs to stay but move to getting it from somewhere else. Just not sure how, if HAPI git repo we are cloning is from a tag then that tag should be the answer. If it is a branch I am not sure. @nathanklick I assume we need this?

Reply:
No think it needs to stay but move to getting it from somewhere else. Just not sure how, if HAPI git repo we are cloning is from a tag then that tag should be the answer. If it is a branch I am not sure. @nathanklick I assume we need this?

Reply:
I put this back in.